lacci/lib/shoes/drawables/font_helper.rb

```diff
@@ -0,0 +1,62 @@
+module Font_helper
+
+    def self.parse_font(font)
```

Comment:
If you want an includable module, you'd normally define these as instance methods (e.g. "def parse_font") rather than self methods. It would be fine to have it free-standing like this and call FontHelper.parse_font, but then you don't need the includes for it. Or you could use the includes, call the methods as "parse_font" rather than "FontHelper.parse_font" and not define them with "self" in front.